### PR TITLE
iMX8QM-MEK flashing instructions

### PR DESCRIPTION
**Pull request by igoropaniuk**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 42012bc2115a6fdea835dd640bed8f07c5b5d642 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1694/docs/artifacts/html/index.html

File: source/reference-manual/boards/imx8qm-mek.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. _ref-rm_boards_imx8qm-mek:
+
+NXP i.MX 8QuadMax Multisensory Enablement Kit (MEK)
+===================================================
+
+.. include:: apalis-imx8-prepare.rst
```
Comment (vanmaegima) [inline source/reference-manual/boards/imx8qm-mek.rst:6]: Should we rename the common file to imx8-prepare.rst?

**Review by MrCry0 — APPROVED** [review]
LGTM (with renaming /apalis-imx8-prepare.rst/imx8-prepare.rst/)

File: source/reference-manual/boards/imx8qm-mek.rst (hunk 1 above)
Comment (igoropaniuk) [reply source/reference-manual/boards/imx8qm-mek.rst]: fixed

**Comment by igoropaniuk** [comment]
All comments addressed

**Comment by doanac** [comment]
Docs for 31b9fbcb563884b97996471a125c1fd1312b4325 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1698/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
This looks perfect on my end.

**Comment by igoropaniuk** [comment]
@kprosise won't you mind if I merge it?

**Pull request merged**